scripts/test_open_manipulator.py

```diff
@@ -0,0 +1,125 @@
+    _pose.position.z = 0.1
+    _pose.orientation.x = 0.0
+    _pose.orientation.y = 0.0
+    _pose.orientation.z = 0.0
+    _pose.orientation.w = 1.0    
+    forward_pose= KinematicsPose()
+    forward_pose.pose = _pose
+    forward_pose.max_accelerations_scaling_factor = 0.0
+    forward_pose.max_velocity_scaling_factor = 0.0
+    forward_pose.tolerance = 0.0
+    try:
+        task_space_srv = rospy.ServiceProxy('/open_manipulator/goal_task_space_path', SetKinematicsPose)
+        resp_delete = task_space_srv("arm","gripper", forward_pose, 2.0 )
+    except rospy.ServiceException, e:
+        rospy.loginfo("Path planning service call failed: {0}".format(e))
+
+def test_rotate():
+    _qpose = JointPosition()
+    _qpose.joint_name = ['joint1','joint2','joint3','joint4']
+    _qpose.position = [0.5, 0.0, 0.0, 0.5]
+    _qpose.max_accelerations_scaling_factor = 0.0
+    _qpose.max_velocity_scaling_factor = 0.0
+    try:
+        task_space_srv = rospy.ServiceProxy('/open_manipulator/goal_joint_space_path_from_present', SetJointPosition)
+        resp_delete = task_space_srv("arm", _qpose, 2.0 )
+    except rospy.ServiceException, e:
+        rospy.loginfo("Path planning service call failed: {0}".format(e))
+    _qpose.position[0] += -1.0
+    _qpose.position[3] += -1.0
+    try:
+        resp_delete = task_space_srv("arm", _qpose, 2.0 )
+    except rospy.ServiceException, e:
+        rospy.loginfo("Path planning service call failed: {0}".format(e))
+    # define actions
+    # assert obs in specific boundary
+
+def test_block_loc():
+    env = OpenManipulatorEnv()
+    for iter in range(20):
+        b_pose =Pose()
+        b_pose.position.x = np.random.uniform(0.15, .20)
+        b_pose.position.y = np.random.uniform(-0.2, 0.2)
+        b_pose.position.z = 0.00
+        b_pose.orientation = overhead_orientation
+        env._load_target_block(block_pose=b_pose)
+        rospy.sleep(2.0)
+        env._delete_target_block()
+    # block generation code
+    # assert block in specific boundary (gripper's movable area)
+
+
+def test_achieve_goal():
+    env = OpenManipulatorEnv()
+    for iter in range(20):
+        b_pose =Pose()
+        b_pose.position.x = np.random.uniform(0.25, .4)
+        b_pose.position.y = np.random.uniform(-0.2, 0.2)
+        b_pose.position.z = 0.00
+        b_pose.orientation = overhead_orientation
+        env._load_target_block(block_pose=b_pose)
+        r_pose = Pose()
+        r_pose.position = b_pose.position
+        r_pose.position.z = 0.08
+        forward_pose= KinematicsPose()
+        forward_pose.pose = r_pose
+        forward_pose.max_accelerations_scaling_factor = 0.0
+        forward_pose.max_velocity_scaling_factor = 0.0
+        forward_pose.tolerance = 0.0
+        try:
+            task_space_srv = rospy.ServiceProxy('/open_manipulator/goal_task_space_path', SetKinematicsPose)
+            resp_delete = task_space_srv("arm","gripper", forward_pose, 2.0 )
+        except rospy.ServiceException, e:
```

Comment:
This line occurs warning in flake8.
I modified as
`rospy.ServiceException as e:`
and above line are executable both on python2.7 and 3.x